client/components/vertical-nav/docs/example.jsx

```diff
@@ -25,13 +27,32 @@ VerticalNavExample.defaultProps = {
 			<VerticalNavItem path="/stats" key="0">
 				Stats
 			</VerticalNavItem>
+
 			<VerticalNavItem path="https://google.com" external key="1">
 				Google
 			</VerticalNavItem>
+
 			<VerticalNavItem path="/posts" onClick={ noop } key="2">
 				Posts
 			</VerticalNavItem>
+
 			<VerticalNavItem isPlaceholder key="3" />
+
+			<VerticalNavItemEnhanced
+				description="Access your emails on the go with our Android and iOS apps"
+				external
+				key="4"
+				materialIcon="smartphone"
```

Comment:
Minor note: this icon is being added in #52780, so is not present in this branch.

Reply:
Good reminder, thank you!